doc/nvim-treesitter.txt

```diff
@@ -74,16 +74,23 @@ Each module corresponds to an entry in the dictionary passed to the
   EOF
 <
 
-All modules share some common options, like `enable` and `disable`.
+All modules share some common options, like `enable`, `disable`, and `cond`.
```

Comment:
Since this hasn't been documented yet, I think would be good to rename this option to be `condition`, `on_condition`, `enable_on` or similar to be more explicit.

Reply:
Sure thing